torch/csrc/profiler/util.cpp

```diff
@@ -32,32 +35,20 @@ std::string getNvtxStr(
       str = name;
 #endif
     }
+    std::stringstream s;
```

Comment:
we avoid `stringstream` when there is no input as it adds significant overhead. This will regress our our no-input collection path

Reply:
removed stringstream, used fmt::format() w/ std::string instead